src/model/GameObject.js

```diff
@@ -225,6 +226,13 @@ GameObject.prototype.scheduleTimer = function scheduleTimer(options, type, key)
 			rc.run(
 				function timerCall() {
 					log.trace({options: options}, '%s call', type);
+					if (self.stale) {
+						if (options.interval) {
+							clearInterval(self.gsTimers[type][key].handle);
+						}
+						delete self.gsTimers[type][key];
+						throw new Error('stale object');
```

Comment:
Is this actually an error state? Is there timeout cleanup code elsewhere that should be doing this? If this is expected I suggest we make this a log.warn and use that to keep track of the places that might need cleanup.